This patch is Debian-specific. Don't use it for non Debian packages. The
program "sensible-browser" is part of the essential package debianutils.

Index: boinc-5.10.0/clientgui/hyperlink.cpp
===================================================================
--- boinc-5.10.0.orig/clientgui/hyperlink.cpp	2006-10-27 12:26:56.000000000 +0200
+++ boinc-5.10.0/clientgui/hyperlink.cpp	2007-05-30 08:59:31.000000000 +0200
@@ -171,7 +171,8 @@
 }
 
 void wxHyperLink::ExecuteLink (const wxString &strLink) {
-    if (!wxLaunchDefaultBrowser(strLink)) {
+    wxString cmd = wxT("sensible-browser ") + strLink;
+    if (!::wxExecute(cmd, wxEXEC_ASYNC)) {
         wxString strDialogTitle = wxEmptyString;
         wxString strDialogMessage = wxEmptyString;
 
